gno.land/cmd/gnoland/testdata/basic_commands.txtar

```diff
@@ -0,0 +1,22 @@
+# test basic gnoland integrations commands
+
+## no arguments should fail
+! gnoland
+
+## should be able to start 
+gnoland start
+
+## should not be able to start a node twice
+! gnoland start
+
+## test1 account should be available on default
+gnokey query auth/accounts/${USER_ADDR_test1}
+
+## invalid gnokey command should raise an error
+! gnokey query foo/bar
+
+## should be able to stop default
+gnoland stop
+
+## should not be able to stop a node twice
+! gnoland stop
```

Comment:
FYI, it's great to have this test, but I believe it should be an exception. We could develop a standard, unique test to enhance the power of the "driver" feature and focus on writing a significant number of real integration tests.

Therefore, feel free to remove this test and the commands specific to it if it helps maintain low complexity or enables the development of new features.

Reply:
I made it exactly for that:
- it's the easiest way for now to test features and basic commands of the driver
- it's also a good example and documentation to know how things works

So yea, i want to keep it for now :)